Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml

```diff
@@ -4,8 +4,21 @@
         <TableSection Title="Crashes Settings">
             <SwitchCell Text="Crashes Enabled" On="true" x:Name="CrashesEnabledSwitchCell" OnChanged="UpdateEnabled" OnColor="{StaticResource SwitchCellOnColor}"  />
             <SwitchCell Text="Handle Exceptions" On="false" x:Name="HandleExceptionsSwitchCell" OnColor="{StaticResource SwitchCellOnColor}" />
-            <TextCell Text="Text Attachment" x:Name="TextAttachmentCell" Tapped="TextAttachment" />
-            <TextCell Text="Binary Attachment" x:Name="FileAttachmentCell" Tapped="FileAttachment" />
+            <ViewCell Tapped="TextAttachment">
```

Comment:
Was that tested on Xamarin? This is shared code.

Reply:
Tested with Xamarin.Forms Puppet iOS and Demo Android.